android/build.gradle

```diff
@@ -44,7 +44,7 @@ allprojects {
 
 dependencies {
     compile 'com.facebook.react:react-native:+'
-    compile 'com.android.support:palette-v7:25.3.1'
+    compile 'com.android.support:palette-v7:23.4.0'
```

Comment:
@rauliyohmc it needs to be compatible with the version of android support library or there'll be unexpected errors.